autogpt_platform/backend/.env.example

```diff
@@ -36,6 +36,15 @@ SUPABASE_JWT_SECRET=your-super-secret-jwt-token-with-at-least-32-characters-long
 GITHUB_CLIENT_ID=
 GITHUB_CLIENT_SECRET=
 
+# Google OAuth App server credentials - https://console.cloud.google.com/apis/credentials, and enable gmail api and set scopes
+# https://console.cloud.google.com/apis/credentials/consent ?project=<your_project_id>
+
+# You'll need to add/enable the following scopes (minimum):
```

Comment:
can we add this informatio into docs?

Reply:
Yeah we don’t really have a good place for that in the docs right now as far as I’m aware. I have a follow up pr I can work it into already open for docs improvements